x-pack/platform/plugins/shared/cases/server/features/v2.ts

```diff
@@ -54,6 +64,9 @@ export const getV2 = (): KibanaFeatureConfig => {
           read: [APP_ID],
           update: [APP_ID],
           push: [APP_ID],
+          createComment: [APP_ID],
+          reopenCase: [APP_ID],
```

Comment:
These should be removed here since they were split out into subfeatures in this version (V2). Adding them in changes `minimal_all` for that version.

`assign` stays because users that had this role before it was split out in V3 could assign cases, so they need to be able to do so still.